src/kafka-to-nexus.cpp

```diff
@@ -24,12 +24,12 @@ void signal_handler(int Signal) {
 int main(int argc, char **argv) {
 
   fmt::print("kafka-to-nexus {:.7} (ESS, BrightnESS)\n"
```

Comment:
Not entirely related to this PR but since there are changes here, this `fmt::print` should be put in the `CLI::App` constructor, no point in having a separate print statement when the constructor takes a string and prints it. 

you can use `fmt::format()` to do this